### DISCUSSION TITLE
Editor reverting all text

### DISCUSSION THREAD

**Discussion by flubber2077** [avitorio/outstatic#271] [Q&A]
I made a basic example project, using vercel and github, (hopefully) following the directions exactly.
logged in on the /outstatic endpoint
made a new post

On the new post editor, I am unable to edit text on seemingly any pre-filled fields(title, author, etc), with them immediately reverting to the default value.

Any help is greatly appreciated, thank you!

**Comment by avitorio** [comment]
Hey @flubber2077 , thanks for flagging this. I just confirmed it is happening and I'm working on a fix.

**Reply by avitorio** [reply]
Hey @flubber2077 ,
This should be fixed on v1.4.14 - you can redeploy the project on vercel and it should work.
Please, let me know if it worked. Thanks!

**Reply by flubber2077** [reply]
![Screenshot 2024-12-10 103005](https://github.com/user-attachments/assets/ceda3db8-af2c-475d-b5a3-40e9f1a0fe50)

![Screenshot 2024-12-10 103115](https://github.com/user-attachments/assets/cbc39ebd-d1cb-438a-a910-42e0b40d788b)

Thank you for the quick response! I updated the package.json with 1.4.14 on outstatic and pushed those changes to github, and then checked vercel (if there is a build step on vercel I am missing please let me know).

On click to make a new post I am given this error, attached are the console errors.

**Reply by avitorio** [reply]
That's weird. What happens if you deploy a new project on vercel following the docs?

**Reply by flubber2077** [reply]
Unfortunately same issue, same error logs

**Reply by avitorio** [reply]
I just tested following this: https://outstatic.com/docs/getting-started#deploy-with-vercel
And it works. Can you please try again?

**Comment by flubber2077** [comment]
My mistake: forgot to update the Oauth address for the test repo I was making. Thank you so much for the quick responses and fix!
